src/core/menu_items/BleMenu.cpp

```diff
@@ -3,7 +3,7 @@
 #include "core/display.h"
 #include "modules/ble/ble_spam.h"
 #include "modules/ble/ble_common.h"
-#include "modules/ble/ble_jammer.h"
+#include "modules/NRF24/nrf_jammer.h"
```

Comment:
I think this line can be removed since you removed the jammer from the options menu